From 16e74f91c93d294156d6c163e38865239223375a Mon Sep 17 00:00:00 2001
From: Greg Kroah-Hartman <gregkh@suse.de>
Date: Wed, 19 Nov 2008 13:59:44 -0800
Subject: [PATCH 14/21] Staging: comedi: comedi_pci.h: remove comedi_pci_enable_no_regions

comedi_pci_enable_no_regions is not called by any comedi code
(in or out of the kernel tree), so remove it, so no one tries to
use it in the future.

Cc: David Schleef <ds@schleef.org>
Cc: Frank Mori Hess <fmhess@users.sourceforge.net>
Cc: Ian Abbott <abbotti@mev.co.uk>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>
---
 drivers/staging/comedi/drivers/comedi_pci.h |    9 ---------
 1 file changed, 9 deletions(-)

--- a/drivers/staging/comedi/drivers/comedi_pci.h
+++ b/drivers/staging/comedi/drivers/comedi_pci.h
@@ -29,15 +29,6 @@
 #include <linux/pci.h>
 
 /*
- * Enables PCI device without requesting regions.  Just a simple wrapper
- * for pci_enable_device().
- */
-static inline int comedi_pci_enable_no_regions(struct pci_dev *pdev)
-{
-	return pci_enable_device(pdev);
-}
-
-/*
  * Called to disable PCI device if PCI device has been enabled, but
  * PCI regions have not been reserved.
  *
